app/controllers/AddAdditionalMetadataController.scala

```diff
@@ -95,17 +95,36 @@ class AddAdditionalMetadataController @Inject() (
       } yield result
   }
 
+  private def getDependenciesOfNonSelectedOptions(dependencies: Map[String, List[FormField]], nameOfOptionSelected: List[String]) =
+    dependencies.filterNot { case (name, _) => nameOfOptionSelected.contains(name) }.flatMap { case (_, fields) => fields.map(_.fieldId) }
+
+  // scalastyle:off cyclomatic.complexity
```

Comment:
We're not using scalastyle any more so you can get rid of this comment. There's a closing one on line 139 which can go as well.

Reply:
IntelliJ uses it (We still have the scalastyle file in the main directory). I still find it kind of useful to have (as it highlights the warnings and makes you think of what your doing and if it could be simplified) but I might remove the Cyclomatic complexity rule (as well as the comment you mentioned) since that is the most annoying warning.

Reply:
It's a bit odd having two formatting tools going at once. I think the best bet is to remove the scalastyle xml file and I think you can then use Intellij to format using scalafmt.

Reply:
Cool, I've removed it